src/textord/cjkpitch.cpp

```diff
@@ -158,11 +158,13 @@ class LocalCorrelation {
     float rc = 0;
     int vote = 0;
     for (int i = start; i < end; i++) {
-      rc += values_[i].vote * x * values_[i].y / values_[i].x;
-      vote += values_[i].vote;
+      if (values_[i].x != 0.0f) {
+        rc += values_[i].vote * x * values_[i].y / values_[i].x;
```

Comment:
@eighttails, did you get a division by zero here, too? Or did it only happen in the return statement?

Reply:
No. I observed error only in the return statement.
This part is to avoid potential error.
I'm not sure if values_[i].x never be zero.

Reply:
I think it would be better to fix only return value right now.